src/directives/layers.js

```diff
@@ -42,14 +42,14 @@ angular.module("leaflet-directive").directive('layers', function ($log, $q, leaf
                     // Only add the visible layer to the map, layer control manages the addition to the map
                     // of layers in its control
                     if (layers.baselayers[layerName].top === true) {
-                        map.addLayer(leafletLayers.baselayers[layerName]);
+                        leafletLayers.baselayers[layerName].addTo(map);
```

Comment:
Well since @elesdoar stated that not all plugins implement addTo then there should be some kind check that takes care of this and uses addLayer if it does not exist.